Documentation/getting-started.md

```diff
@@ -0,0 +1,143 @@
+download URL. For this example we'll just be running it off of the local
+filesystem, so name it whatever you want.
+
+```bash
+acbuild set-name example.con/nginx
+```
+
+For more information on meta discovery, check out the [appc
+spec](https://github.com/appc/spec/blob/master/spec/discovery.md#meta-discovery).
+
+### Adding a dependency
+
+Having an empty ACI isn't all that useful, so let's add a dependency.
+
+```bash
+acbuild dependency add quay.io/coreos/alpine-sh
+```
+
+Now when this ACI gets run the image at `quay.io/coreos/alpine-sh` will be
+fetched, and used as a base for our image. This means that our ACI only needs
+to contain the files that we add or modify on top of alpine.
+
+### Installing nginx
+
+Now that we've got a base image with fancy things like a shell and a package
+manager, let's install nginx.
+
+```bash
+acbuild run -- apk update
+acbuild run -- apk add nginx
+```
+
+The first command updates alpine's package manager, and the second command
+fetches and installs nginx. The `--` in each command stops acbuild from trying
+to parse flags that occur after the `--`. They're not necessary in this
+instance, but they would be if one of our commands had any arguments that began
+with a `-`.
+
+### Adding a mount point
+
+Since the goal of this is to host a static website, we're going to add a mount
+point to the default location that nginx serves files out of. This can be used
+when the ACI is run to make a directory on the host available inside of the
+container.
+
+```bash
+acbuild mount add html /usr/share/nginx/html
+```
+
+### Setting the exec statement
+
+The last thing we need to set in our ACI is the executable we want it to run.
+Without this command, the ACI couldn't be used without specifying a path to an
+executable inside of it at runtime.
+
+```bash
+acbuild set-exec -- /usr/sbin/nginx -g "daemon off;"
+```
+
+The `--` argument serves the same purpose here as it did when we were using the
+`run` command, except it actually matters this time. Without it, acbuild would
+think that the `-g` argument was a flag for it, instead of something to pass to
+`/usr/sbin/nginx` when the ACI is run.
+
+### Writing out the ACI
+
+With that, our ACI is complete. Before ending the build however, we need to
+write out the ACI to a file.
+
+```bash
+acbuild write nginx.aci
+```
+
+### Ending the build
+
+This command will delete our current build context. Had we not written out an
+ACI, everything we've done up to this point would be lost.
+
+```bash
+acbuild end
+```
+
+## Using the ACI
+
+If you have a directory with some files for nginx to serve...
+
+```bash
+mkdir test
+echo "Hello, world!" > test/index.html
+```
+
+... using the ACI is something as simple as the following command:
+
+```bash
+rkt run --insecure-skip-verify ./nginx.aci --volume html,kind=host,source=/path/to/test --net=host
+```
+
+Now point your browser at [`http://localhost:8888`](http://localhost:8888).
```

Comment:
On my computer, nginx started on port 80 and not 8888. So it should just be `http://localhost/` or `http://localhost:80/`.

Also, I get a "403 Forbidden" error, and strace on nginx says:
```
open("/usr/share/nginx/html/index.html", O_RDONLY|O_NONBLOCK) = -1 EACCES (Permission denied)
```

Indeed, the process nginx runs as user `100` (and not `root`) and it does not have `x` permission on `/`:
```
# ls -ld  /proc/`pidof 'nginx: worker process'`/root/
drw-r--r--. 1 root root 4096 Nov 14 16:24 /proc/970/root/
```

This comes from the alpine image, as reported on https://github.com/coreos/rkt/issues/1703#issuecomment-152848856: 
```
$ wget -O - https://quay.io/c1/aci/quay.io/coreos/alpine-sh/latest/aci/linux/amd64/ | tar tvz|head
drw-r--r-- 0/0               0 1970-01-01 01:00 rootfs/
```